src/app/admin/admin.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="main">
+  <p-table [value]="molecules">
+    <ng-template pTemplate="header">
+      <tr>
+        <th>Molecule ID</th>
+        <th>Name</th>
+        <th>User</th>
+        <th>Group</th>
+        <th>Block Set ID</th>
+        <th>Block IDs</th>
+        <th>Actions</th>
+      </tr>
+    </ng-template>
+    <ng-template pTemplate="body" let-molecule>
+      <tr>
+        <td>{{ molecule.id }}</td>
+        <td>{{ molecule.name }}</td>
+        <td>
+          {{ molecule.created_by.name }} ({{ molecule.created_by.username }})
+        </td>
+        <td>{{ molecule.submitted_to?.name }}</td>
+        <td>{{ molecule.block_set_id }}</td>
+        <td>{{ molecule.block_ids.join("-") }}</td>
+        <td>
+          <p-button label="Synthesize" (onClick)="synthesize(molecule.id)" />
```

Comment:
it appears from the source code that this button is always displayed. is that right? just checking

would it be possible to display whether each row has already been sent to the synthesizer (or even a more detailed status/history)?

these aren't requests that have to be addressed in this PR